requirements/pytorch/loggers.info

```diff
@@ -1,5 +1,5 @@
 # all supported loggers. this list is here as a reference, but they are not installed in CI
 neptune-client
 comet-ml
-mlflow
+mlflow>=1
```

Comment:
```suggestion
mlflow>=1.0.0
```

Reply:
Remember that we dont install these dependencies in CI. If our logger implentation does not work in lt 1.0, then it should be checked in the code at init time. For example by using RequirementCache

Reply:
Ok, I took a stab at this: https://github.com/Lightning-AI/lightning/pull/15915/commits/3428e5dbd6b2554dd2506b0d67c54f94569f5aa8

Let me know what you think!